lte/gateway/c/core/oai/tasks/nas5g/src/ies/M5GSMobileIdentity.cpp

```diff
@@ -194,6 +194,35 @@ int M5GSMobileIdentityMsg::DecodeImsiMobileIdentityMsg(
 
   MLOG(MDEBUG) << "Length :  " << int(ielen);
   memcpy(&imsi->scheme_output, buffer + decoded, ielen - decoded);
+
```

Comment:
is `DecodeImsiMobileIdentityMsg` covered by any unit tests?

Reply:
Yes @ulaskozat, covered as part of UT test_amf_ue_suci_ext_register_req_msg in test_amf_encode_decode.cpp